src/MSBuild/LiveLogger/TerminalBuffer.cs

```diff
@@ -200,6 +211,27 @@ public static int GetLineIndexById(int lineId)
             return line;
         }
 
+        public static TerminalBufferLine? WriteNewLineAfterMidpoint(string text, bool shouldWrapLines = false)
```

Comment:
I don't think I grok these new methods. What's the significance of the midpoint?

Reply:
They are fully unnecessary for the change as-is, but the midpoint separated the projects being built from the blocked projects lists. In commenting out the blocked projects part, I left those in to try to minimize the magnitude of the change.